packages/dds/tree/src/test/feature-libraries/editableTree.spec.ts

```diff
@@ -0,0 +1,511 @@
+/*!
+* Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+* Licensed under the MIT License.
+*/
```

Comment:
Formatting got messed up here causing the repo policy check to fail:

```suggestion
/*!
 * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
 * Licensed under the MIT License.
 */

```